XML/TestCases/FunctionalTests-TVM.xml

```diff
@@ -26,10 +27,11 @@
     <Tags>measuredboot</Tags>
     <Priority>1</Priority>
     <SetupConfig>
-      <TestLocation>eastus2euap</TestLocation>
+      <TestLocation>southcentralus</TestLocation>
```

Comment:
Can you please remove this default location completely?

Rest looks fine!

Reply:
Discuss internally, keep it as is.